### PR TITLE
Fixes #13975: Add compatibility sles classes in failsafe

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/13975

### CODE REVIEW

**Comment by ncharles** [comment]
you hsould not change path of apache-vhost.tpl

File: techniques/system/common/1.0/failsafe.st [hunk 1]
```diff
@@ -126,6 +126,12 @@ bundle common g
 
 # definition of the node roles
   &NODEROLE&
+
+  classes:
+      # OS classes for compatibility
```
Comment (peckpeck) [inline techniques/system/common/1.0/failsafe.st]: shouldn't there be a scope => namespace ?

Comment (amousset) [reply techniques/system/common/1.0/failsafe.st]: Nope, it is a bundle common where classes are global by default.

**Comment by amousset** [comment]
ping @peckpeck

**Pull request merged**